src/algorithms/calorimetry/ImagingPixelReco.h

```diff
@@ -36,7 +36,7 @@ class ImagingPixelReco {
     // length unit (from dd4hep geometry service)
     double m_lUnit; // {this, "lengthUnit", dd4hep::mm};
     // digitization parameters
-    unsigned int m_capADC; // {this, "capacityADC", 8096};
+    unsigned int m_capADC; // {this, "capacitysize_tADC", 8096};
```

Comment:
```suggestion
    unsigned int m_capADC; // {this, "capacityADC", 8096};
```

Reply:
The disadvantage of using the middle mouse button for both scrolling and pasting...